backend/main.py

```
@@ -0,0 +1,550 @@
+import os
+import glob
```

Ruff (0.12.2)

`glob` imported but unused

Remove unused import: `glob`

(F401)